app/assets/stylesheets/application.css.scss

```diff
@@ -15,3 +15,7 @@
  *= require foundation_and_overrides
 
  */
+
+ @import 'modules/topbar';
```

Comment:
Line should be indented 0 spaces, but was indented 1 spaces<br>Prefer double-quoted strings